doc/west/index.rst

```diff
@@ -1,30 +1,142 @@
 .. _west:
 
-West (Experimental)
-###################
+West
+####
 
-The Zephyr project is developing a swiss-army knife command line tool
-named ``west``. (Zephyr is an English name for the Greek god of the
-west wind.)
+The Zephyr project includes a swiss-army knife command line tool
+named ``west`` (Zephyr is an English name for the Latin
+`Zephyrus <https://en.wiktionary.org/wiki/Zephyrus>`_, the ancient Greek god
+of the west wind).
 
-West is developed in its own `repository on GitHub`_. A copy is
-currently maintained in Zephyr's :file:`scripts/meta/west` directory.
+West is developed in its own `repository on GitHub`_.
+
+West is used to obtain the source code for the Zephyr project and can also be
+used to build, debug, and flash applications.
+
+History and motivation
+**********************
+
+West was added to the Zephyr project to fulfill two fundamental requirements:
+
+* The ability to work with multiple Git repositories
```

Comment:
I know we talked about this already, but for the benefit of everybody;
Why was `repo` not deemed fit for working with multiple Git repositories?

Reply:
In short:

* Python 2 only
* Uses symlinks so it doesn't work on windows without an admin command prompt
* Hardcoded gerrit code review
* Poorly documented and maintained

Reply:
I think these repo shortcomings are important to mention somewhere in the documentation itself and not just in this code review, otherwise west-multirepo looks like just another NIH tool.

While adding them drop "poorly documented and maintained" or justify it a bit https://gerrit.googlesource.com/git-repo/
https://gerrit.googlesource.com/git-repo/+log/master

Repo is used by a fair number of projects despite these shortcomings. So you are indirectly saying the world could use a better multirepo tool. I could agree. 
 => Why should [west-]multirepo be specific to Zephyr?